Docs audit for credentials section (#317)
KILTprotocol/docs: docs/concepts/05_credentials/01_overview.md

/>
</center>

> To get a credential, a Claimer needs to take the following process:

Reviewer: ```suggestion
To get a credential, a Claimer needs to take the following steps:
```

For me `steps` feels more natural. But not a big deal.

1. Find a **CType** to base a claim on. Potential Attesters and Verifiers might advertise this information themselves.
2. Make a **claim** containing a set of properties about themselves.

> 3. Optionally request and receive **Terms** and agree on a **Quote** with the potential Attester.

Reviewer: ```suggestion
3. Fulfil any requirement from your Attester; e.g. accepting their **Terms** and paying a **Quote**. 
```

Fromthe `Claimers` perspective, it is not really _**optional**_. 

If you just don't want to repeat "**_Potentially_**", and don't like my suggestion above: how about just "**_Possibly_**"?

4. **Request an attestation** from the Attester.
5. Wait for the Attester to **attest** claims.

> Once attested, the claims are considered to be a valid credential.

Reviewer: I'm not sure if this statement is 100% true. 

I think there is wrapper around the claims that includes the cType and a Hash of the whole thing. Only wrapped it is considerd a credential. 

I will like to check this tomorrow.

Reviewer: I know this was not changed on this branch. But it would be nice to fix, if it's not truthful.

Reviewer: Example Credential: 
```
{
      "claim": {
         "cTypeHash": "0x3291bb126e33b4862d421bfaa1d2f272e6cdfc4f96658988fbcffea8914bd9ac",
         "contents": {
            "Email": "refij28030@in2reach.com"
         },
         "owner": "did:kilt:4tFve5tV8dA8hQ74Q2jFYB8sep7HS2Mi4Zdm9DFa25rtiu5E"
      },
      "claimHashes": [
         "0x68176b55844c0645a0bd778cafb67185581cdb32106b89a8691d29b8402e37cd",
         "0xf8b622c16489ddf7bd601bbbbe69e3e5e865341ac001d08816a89c977fa2a4e7"
      ],
      "claimNonceMap": {
         "0x702ce98117a4772b27547e61b6ab035da2c2947852c933c1888e0c40bce47f75": "8ae79e6a-ff2f-4a8c-8061-b2d33243d006",
         "0xec2a9c4ed9575eb5024d93d9c7d2a08430d9a642e6da067386d091f2b419c2ac": "df741c65-ba1d-4b21-9b61-0fae5e2caa45"
      },
      "delegationId": null,
      "legitimations": [],
      "rootHash": "0x87d4140cf379e201746567bb4c1aad025965255362d66ea816104219b3e188e4"
   }
```

Reviewer: So, the claims are just a (core) part of the credential.

> To verify a credential, a Claimer can generate a presentation of it to a Verifier, with the following process:

Reviewer: ```suggestion
To use a Credential, the Claimer can generate a Credential-Presentation for a Verifier.
The verification would follow this process:
```

> 1. The Verifier may request a **credential** as the first step, along with with properties to reveal from the Credential.

Reviewer: ```suggestion
1. The Verifier may request a **Credential** of a CType, along with with properties to reveal.
He would also provide a **challenge** to ensure the presentations are not recycled. 
```

> 2. The Claimer selectively **discloses** the requested properties and signs the generated presentation.

Reviewer: ```suggestion
2. The Claimer selectively **discloses** the requested properties and signs them along with the challenge to generate a presentation.
```

> 3. The Verifier **checks** the presentation structure, content and signature, and decides whether they trust the Attester of the presented credential.

Reviewer: ```suggestion
3. The Verifier **verify** the presentation structure, content and signature, and decides whether they trust the Attester of the presented credential.
```

If it on **bold**, I would use the official term.

/>
</center>

To get a credential, a Claimer needs to take the following steps:

1. Find a **CType** to base a claim on. Potential Attesters and Verifiers might advertise this information themselves.
2. Make a **claim** containing a set of properties about themselves.
3. Fulfil any requirement from your Attester. For example, accepting their **Terms** and paying a **Quote**.
4. **Request an attestation** from the Attester.
5. Wait for the Attester to **attest** claims.

> Once attested, the claims are recorded on chain and considered to be a valid credential.

Reviewer: The claims are never recorded on chain! 

Only the root hash of them. 

```suggestion
Once attested, the wrapped claims are considered to be a valid credential.
```
